runtime/testnet/src/config/xcm.rs

```diff
@@ -171,34 +174,34 @@ pub type XcmRouter = WithUniqueTopic<(
 )>;
 
 impl pallet_xcm::Config for Runtime {
+	type AdminOrigin = EnsureRoot<AccountId>;
+	// ^ Override for AdvertisedXcmVersion default
```

Comment:
Not related to this PR actually, but we miss a dot :o 
```suggestion
	// ^ Override for AdvertisedXcmVersion default.
```